packages/palette/src/elements/GridColumns/GridColumns.story.tsx

```diff
@@ -0,0 +1,176 @@
+    left={0}
+    width="100%"
+    height="100%"
+    style={{ pointerEvents: "none" }}
+  >
+    {[...new Array(12)].map((_, i) => (
+      <Column
+        key={i}
+        columnSpan={[1]}
+        bg="rgba(255, 0, 0, 0.05)"
+        height="100%"
+      />
+    ))}
+  </GridColumns>
+)
+
+storiesOf("Components/GridColumns", module)
+  .add("Real-world Example", () => {
+    return (
+      <GridColumns position="relative" m={2}>
+        <GridColumnsDebug />
+
+        <Column columnSpan={6} wrap>
+          <Text as="h1" variant="largeTitle">
+            Page Title Long Enough So As To Line Break
+          </Text>
+        </Column>
+
+        <Column columnSpan={6}>
+          <Text as="h2" variant="subtitle" mb={1}>
+            Page subtitle
+          </Text>
+
+          <Text>{IPSUM}</Text>
+        </Column>
+
+        <Column columnSpan={5} columnStart={8}>
+          <Text variant="mediumText" mb={1}>
+            Featured content
+          </Text>
+
+          <Box bg="black10" borderRadius={4} height={400} />
+        </Column>
+      </GridColumns>
+    )
+  })
+  .add("Kitchen sink", () => {
+    return (
+      <GridColumns border="1px solid blue" position="relative">
+        <GridColumnsDebug />
+
+        <Column border="1px solid red" columnSpan={4}>
+          <Text>{IPSUM}</Text>
+        </Column>
+
+        <Column border="1px solid red" columnSpan={4}>
+          <Text>{IPSUM}</Text>
+        </Column>
+
+        <Column border="1px solid green" columnSpan={4}>
+          <Text>{IPSUM}</Text>
+        </Column>
+
+        <Column border="1px solid red" columnSpan={4} wrap>
+          <Text>
+            {IPSUM} {IPSUM}
+          </Text>
+        </Column>
+
+        <Column bg="black100" columnSpan={[6]}>
+          <Text color="white100">
+            This remains 2-columns at all breakpoints
+          </Text>
+        </Column>
+
+        <Column bg="black100" columnSpan={[6]}>
+          <Text color="white100">
+            This remains 2-columns at all breakpoints
+          </Text>
+        </Column>
+
+        <Column bg="black100" columnSpan={6}>
+          <Text color="white100">
+            These are 2-columns at large breakpoints, and 1 at mobile
+          </Text>
+        </Column>
+
+        <Column bg="black100" columnSpan={6}>
+          <Text color="white100">
+            These are 2-columns at large breakpoints, and 1 at mobile
+          </Text>
+        </Column>
+
+        <Column
+          columnStart={[2, 3, 4]}
+          columnSpan={[9, 6, 3]}
+          height={300}
+          bg="black10"
+          borderRadius={4}
+        />
+
+        <Column
+          height={200}
+          bg="purple100"
+          columnSpan={[12, 6]}
+          columnStart={[1, 4]}
+          wrap
+        />
+
+        <Column bg="black100" columnStart={4}>
+          <Text color="white100">{IPSUM}</Text>
+        </Column>
+
+        <Column border="1px solid red" columnSpan={4} columnStart={1}>
```

Comment:
Talked offline, but woke up thinking about this props name thing and, i think brevity is probably better, since `Column` is in the component name. Sorry for the back and forth!

Reply:
I was also flipping back and forth depending on what ultimately this was named. like `Cell columnSpan` makes sense and `Column span` makes sense. Since we're going with `Column` for now, we'll go with that. Always can revisit if it proves confusing.